dataplane/Cargo.toml

```diff
@@ -5,6 +5,10 @@ edition = "2021"
 publish = false
 license = "Apache-2.0"
 
+[[bin]]
+name = "gateway"
+path = "src/main.rs"
```

Comment:
This will require some additional changes in the `Dockerfile`, as well as in the `justfile` for targets `sterile-build` and `build-container` where we reference binaries or Rust packages called `dataplane`.